### ISSUE TITLE
Election should say "ended" if you visit it after it ends

### ISSUE THREAD

**Issue by craigbox** [elekto-io/elekto#79]
The election is marked "completed", so the tenses of the start/end time listed at the bottom should be "started" and "ended"

<img width="949" alt="Screen Shot 2022-10-17 at 1 36 48 PM" src="https://user-images.githubusercontent.com/132510/196067001-d621f79b-ae36-4286-928e-6a290ca5bdbc.png">

**Comment by jberkus** [comment]
I don't follow?

**Comment by bannmann** [comment]
@jberkus Just stumbled over this. I think what OP meant was that in the screenshot, the last line reads:

> Voting starts at 2022-10-03 UTC and ends at 2022-10-16 11:59:59 UTC.

However, the screenshot was clearly made after the election was over (as it says "Completed" below the heading). So "starts at" and "ends at" are confusing. So for completed elections, the text should change as follows:

> Voting ~~starts~~***started*** at 2022-10-03 UTC and ~~ends~~***ended*** at 2022-10-16 11:59:59 UTC.

**Comment by jberkus** [comment]
Makes sense.  That seems like a good small change for a new contributor, so reopening.

**Comment by justaugustus** [comment]
+1. Just ran into confusion with this one.

**Comment by justaugustus** [comment]
Additionally, from the perspective of an election officer, it is unclear if "Completed" means “The logged-in user has finished voting” or “The voting window has concluded”?
